### ISSUE TITLE
Friendly error links have poor color contrast

### ISSUE THREAD

**Issue by stampyzfanz** [processing/p5.js-web-editor#2046]
#### Details about the bug:

- Web browser and version: Firefox 102.0.1
- Operating System: Windows 10
- Steps to reproduce this:
https://editor.p5js.org/stampyzfanz/sketches/flp7cojOM
![image](https://user-images.githubusercontent.com/34364128/181870774-70b47ac6-b7f8-4527-903f-96a1686bacc2.png)
The problem is inline CSS that changes the colour of FES links without taking into account the theme that is used.

**Comment by Brady-Edwards** [comment]
Hi there, if someone isn't already working on this, I'd like to have a crack! This would also be my first open source contribution :).

**Comment by stampyzfanz** [comment]
I am not aware of anyone working on it and feel free to let me know if there's anything I can do to help.

Thanks for your interest :)

**Comment by Brady-Edwards** [comment]
I've got a working fix, can a maintainer give me the green light to open a PR? Thanks.

**Comment by raclim** [comment]
Hi @Brady-Edwards, super excited to see your work! :) Please feel free to go through with the PR, or let me know if there's anything else you need beforehand!

**Comment by devtarun** [comment]
Hello. is this still open?

**Comment by ayushbhaimehta** [comment]
Hi there. If this haven't been resolved yet then I would love to do so. Also can somebody point me where can I get slack or discord  link where there is more discussion on issues. 
Thank you

**Comment by raclim** [comment]
Hi @devtarun and @ayushbhaimehta! Yes, this is still currently open, but I think @Brady-Edwards had some work done on this earlier. However, please feel free to add to this issue! 

This is the [invite URL for the p5.js discord](https://discord.gg/SHQ8dH25r9). :)

**Comment by Brady-Edwards** [comment]
Hi there, I had completely forgotten about this (got carried away with life)! I had already pin-pointed the issue and a fix for it but never got around to issuing a pull request. However, I'm more than happy for someone else to issue the pull request for this instead :). Either way, I'll quickly detail what I found to cause the issue here:

In this snippet [here](https://github.com/processing/p5.js-web-editor/blob/c6f2936600497c677b0a8c01aa55946d1b5be899/client/styles/abstracts/_placeholders.scss#L171-L187), the `color` and `fill` lines are getting overridden with some external css (not sure what's the cause for the overwrite, but it doesn't seem to be from a css stylesheet in the repository). To prevent these from being overridden, the `!important` tag can be used, which seemed to fix the bug for me. I'm sure there's a better fix out there, but this is all I could find from the time I spent looking at this.

**Comment by ayushbhaimehta** [comment]
@raclim I pretty much agree with brady. The CSS seemed to be overridden by 2-3 external modules of CSS (one is definitely from main but still can't figure out the other ones).

**Comment by lindapaiste** [comment]
I looked into this a bit so I want to get this written down for posterity.  The TL;DR is that we are missing an important prop on a component from an external package.

Here's what I'm seeing in dev tools 
![image](https://user-images.githubusercontent.com/28965286/219904826-c7917db7-7d48-4989-9ca9-d7c1641d9688.png)

There is an appropriate link color `#f0f0f0`  that is set for all links when in `.contrast`, but that gets overwritten by another style which sets the link color for the console.  That's the Emotion-generated CSS `.css-kidsne a`.  The console element `<div data-type="string" class="css-e393sm">` is a closer ancestor than the `<body class="contrast">` so that style will get priority.

The console component comes from an external package `'console-feed'`, but we are passing in [quite a lot of styling information](https://github.com/processing/p5.js-web-editor/blob/cb331c7db1ff17796254e7eac52b1f2c5aa97b5a/client/modules/IDE/components/Console.jsx#L40-L202) to customize it.

What we are **not** doing, however, is passing a `variant` prop to the `<ConsoleFeed/>` component!  The `console-feed` package supports `light` and `dark` modes, and the default styling will change based on which you select. You can see that in their source code [here](https://github.com/samdenty/console-feed/blob/b56a00a7cbad4caaac0c221d328473bbf9f17543/src/Component/theme/default.ts#L27):
```
LOG_LINK_COLOR: isLight ? 'rgb(66, 66, 66)' : 'rgb(177, 177, 177)',
```

There are two ways to fix this:
1. We can pass a `variant="dark"` prop to the `<ConsoleFeed/>` when in "dark" or "contrast" modes.  This changes the default color from the unreadable dark gray `rgb(66, 66, 66)` to a light gray `rgb(177, 177, 177)`

Contrast:
![image](https://user-images.githubusercontent.com/28965286/219905284-133a3457-f1a2-4b09-bb2b-c49aedf71f4b.png)
Dark:
![image](https://user-images.githubusercontent.com/28965286/219905328-2ea94cb2-c694-4cb6-8253-e35eff361d1b.png)

We need to add one additional line to the code [here](https://github.com/processing/p5.js-web-editor/blob/cb331c7db1ff17796254e7eac52b1f2c5aa97b5a/client/modules/IDE/components/Console.jsx#L269-L273)
```
<ConsoleFeed
    variant={theme === "light" ? "light" : "dark"} // <---- this line
    styles={getConsoleFeedStyle(theme, fontSize)}
    logs={consoleEvents}
    key={`${theme}-${fontSize}`}
/>
```

2. We can define our own `LOG_LINK_COLOR` as whatever precise color we want it to be.  We would add an additional property to the [`CONSOLE_FEED_DARK_STYLES`](https://github.com/processing/p5.js-web-editor/blob/cb331c7db1ff17796254e7eac52b1f2c5aa97b5a/client/modules/IDE/components/Console.jsx#L69) and [`CONSOLE_FEED_CONTRAST_STYLES`](https://github.com/processing/p5.js-web-editor/blob/cb331c7db1ff17796254e7eac52b1f2c5aa97b5a/client/modules/IDE/components/Console.jsx#L105) objects.

I recommend #1 for simplicity.
